### ISSUE TITLE
Сервомотор поворачивается рывками

### ISSUE THREAD

**Issue by Olejan** [AlexGyver/GyverDrink#13]
Сервомотор поворачивается рывками, при этом разливая жидкость из трубки на поверхность наливатора. А в сервисном режиме, если крутануть резко энкодер, то серву начинает дёргать туда-сюда, пока она не доедет до заданной позиции. Использовал в проекте библиотеку ServoSmooth из архива, и новую - результат не меняется, только с новой библиотекой серво поворачивается медленней.
Игрался с настройками скорости и ускорения - не помогает, всё равно поворачивает рывками и разбрызгивает жидкость по наливатору. Питание нормальное. Помпа потребляет 0,5 А - батарея тянет. Серво потребляет в несколько раз меньше.
Есть какие-то рекомендации по исправлению эфекта дёргания сервы? Понимаю, раз у вас работает, должно и у меня работать - ан нет.

На видео версия, адаптированная под двухстрочный индикатор.
Настройки серво из видео:
	// настройка серво
	,,,
	servoON();
	servo.attach(SERVO_PIN, 200/*600*/, 2800/*2400*/);
	if (INVERSE_SERVO) servo.setDirection(REVERSE);

	servo.write(0);
	delay(800);
	servo.setTargetDeg(0);
	servo.setSpeed(40);
	servo.setAccel(0.3);

https://user-images.githubusercontent.com/5707511/209798714-3084589e-ccb2-40e0-afc7-65a30ce1fccc.mp4

**Comment by GyverLibs** [comment]
Очень странно, может серва бракованная

**Comment by Olejan** [comment]
Да, действительно. Заменил серву и уменьшил диаметр трубки стяжкой - эффект разбрызгивания ушёл.
Прошу прощения за беспокойство и с наступающим Новым Годом!

**Comment by GyverLibs** [comment]
С наступающим да)
